bukuserver/views.py

```diff
@@ -135,7 +135,7 @@ def _list_entry(self, context: Any, model: Namespace, name: str) -> Markup:
             res += [f'<span class="netloc"> ({link(netloc, url_for_index_view_netloc)})</span>']
         if not parsed_url.scheme:
             res += [f'<span class="link">{escape(model.url)}</span>']
-        elif self.url_render_mode is None or self.url_render_mode == 'full':
+        elif self.url_render_mode == 'full':
```

Comment:
No change in logic here; this property is actually implemented in a way that prevents returning `None` in the first place